upstream-community-operators/kfctl/0.1.0/kfctl.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,107 @@
x+z3VTV/elQ2zi5OzmKMmXKIn5c59Pzm76X/S2BYISD/brel2gAwAcBAUeANB7s8noKu18MPrroXSAQh6auurDKdPPcUUr3+5yQCXef8T7F+zDQ5bjPryfAgA9osADAIbCSQb65jntnMdOW1+JeFk6B51zeXJ28/P69GZ/tXO43kaJB7v0nB68nwIA/aPAAwAGYTYZ/bIICn1x0dRVp0vpk7Ob07Sn7+BHnCb5OqASb1E4CfTRS5KL9c8ZAMBBUeABAIMxm4yeksxL5+iwcekA/Ld5U1edPpGzLl2+ls5B550m+XldBvfXcvyS5XgeJR5s29X6uloAgIOjwAMABmU2GS1iAfRH9fuUS3csmrpalA6xBV/j7xTbcZz2JF6/S7wk6xLvU+kY0BPXWY4XpUMAAPweBR4AMDizyWge84Topqemrjp/ivTk7OZz2pNTsC1HaUu8d6WD7Nxy/DFOk8OmHrIcX5UOAQD8/+zdQVIbeb4u7JcTd6iIw7eCy92AhBZANDA9g4YVCCY57DZDRnaNNMSlIRPUK7DPQFPBibMA072B4qzgciOY9zfIpMpV7SobI+mvzHyeCIerXC7xGgTC+ebv/+OPKPAAgL46Sr33hG7oQyFb7+lpueF4dpbkTekcdNJukg/Nc6zb6qmh83gdg+/xkA68ngIA3afAAwB6aXKw85i6xKMDltNBHy5iny6ng4fSIV6jOeLwqnQOOu9mOJ51vySuSzw3o8DL1DfDLEY+bwCArafAAwB6a3Kwcx/HkNEOF8vp4K50iNcYjme7ST7E3js242o4nt2UDrF2i9F96hLvoXASaIuL5vMGAGDrKfAAgF6bHOzMk8wLx2iLvdIBeurjcjp4XzrECtzEc4jNOhuOZzdNedxddRkxTj+OEobXeN9MrgIAtIICDwDovcnBznlc+PwWe6UD9FAnpkSH49m7JCelc9BLZ0lue1DiPR8L7bUMvuwui9FF6RAAAC+hwAMAqNkjxLZ5THLe9v1+w/HsJMnb0jnotf30pcRbjMYxVQ6/9ZDktHQIAICXUuABACSZHOw8Ty/QXl2bPDlfTget/jMNx7O91EdnQmn7SX4ajmf7pYOs3WJ0HiUePHtMctpMqQIAtIoCDwCgMTnY6cRxhT3WpYtz75fTwcfSIV6jmXb6kKTbU0+0yW7qSby+lHiOC4TkotkTCQDQOgo8AIDPTA525jG5QFl3y+mgCxfer1JPPcE2eS7xzkoHWbvF6H3clEK/vc9iNC8dAgDgeynwAAB+Y3Kwc57uHce4En/773+aplqvh3RgT89wPHuT5Kx0Dvgdu0luelLizVN/TenShDJ8i7ssRl24GQYA6DEFHgDAlx3FBc8vMVG1XqfL6aDVz7vmeMKr0jngG9wMx7N3pUOs3WL0MV7T6Jd67x0AQMsp8AAAvmBysPOY+oInbMr5cjpo9eRns/futnQOeIG3w/HspnSItat3gB2lnvKFrjvKYqSwBgBaT4EHAPA7Jgc797E/qE0eSgd4hflyOpiXDrECt6mPJ4Q2ORuOZzdNAd1ddYk3jiOi6bbz5rkOANB6CjwAgD8wOdiZJ5kXjsG3+Z/SAb7T/XI6aH1RPBzPruKIVdrrLMltD0q85+lyBQddNG/2PgIAdIICDwDg6y7iYifr0Yk9PcPx7CzJm9I54JX205cSbzEax80pdMt9FqPW3wwDAPA5BR4AwFc0+/BOU5ctsEqny+ngoXSI1xiOZ/tJrkrngBXZT/JT87zutrrsmJeOAStgbzEA0EkKPACAbzA52HlIByalVuCwdIAOuVhOB3elQ7xGM6n0Ifbe0S27qSfx+lLiXZSOAa901BwPCwDQKQo8AIBvNDnYuYsLnazGx+V08L50iBW4SbJXOgSswXOJd1Y6yNotRu+TOHqQtjrPYuSYcwCgkxR4AAAvMDnYeR9Hjm2rttx9f58OXCwfjmfvkpyUzgFrtJvkpicl3jyOiqZ95s1zFwCgkxR4AAAvd5G6hGG7tOFj8pjkfDkdtPoi+XA8O0nytnQO2JCbprDutsXoY+o9Yq3++kRv3DdHwAIAdJYCDwDghSYHO48xqcD3OV9OB20oGn/XcDzbS310JvTJ2+F41v3nfX0U4VHacUME/fWY+nkKANBpCjwAgO8wOdh5SF3iwbd6v5wOPpYO8RrD8Ww3yYfURwtC35wNx7MPzedBdynx2H5HWYzcRAUAdJ4CDwDgO00Odu5SH6fZJ38qHaCl7pbTQReeK1dJ9kuHgIJOktz2oMR7nnC6K5wEfuu8KZkBADpPgQcA8AqTg533Sealc7DVHtKBac3hePYmyVnpHLAF9lOXeHulg6zVYvSYxegoXuPYHvMsRvPSIQAANkWBBwDwehdx1Bi/73Q5HbT6qK/heLafevoOqO0n+dR8bnTbYnQeJR7l3TfPRQCA3lDgAQC80uRg5zH1hFWrS5q2W04Hd6UzfMH5cjpodbnbHBV4WzoHbKHd1JN4fSnxlCeU8nykKwBAryjwAABWYHKw85AOHJPISs2X08G8dIgVuE1dVAD/ajf1JN5Z6SBrVx9dqMSjhKNmLyMAQK8o8AAAVmRysHOX+jhNuF9OB62/0D0cz65SHxUI/LGbHpV4RzFxzuacZzFq9SQ7AMD3UuABAKzQ5GDnfbq9K0iZ83XPR6q2WlNGvCmdA1rkpim9u20xuosSj82YN6UxAEAvKfAAAFbvIklX7xZ3lOLXnS6ng4fSIV6j2enV/SICVu/NcDy7KR1i7eqJqKN097WO8u6b3YsAAL2lwAMAWLHJwc7zBJbphP65WE4Hd6VDvMZwPNtN8iHKWvheZ8Px7EPzudRdSjzWpxOT7AAAr6XAAwBYg8nBzkNcfCrhruDb/ricDt4XfPurcpNkr3QIaLmTJLc9KPEeU5d4d4WT0C2nWYweSocAAChNgQcAsCaTg5271Mdp0n33SVp/1NdwPHuXungAXm8/dYm3VzrIWi1Gj1mMjtLt/a9szkWzZxEAoP
```

Comment:
'replaces' property is pointing to non-existing operator kfctl.v0.0.0, please remove the line